### ISSUE TITLE
Make list of studies dependent on selected species

### ISSUE THREAD

**Issue by Aryllen** [Sage-Bionetworks/dccvalidator#163]
Copy/paste from Bug Bash document since it gives more of the thought process for this suggestion.
> Put species before study since that determines what studies are in the list. Never mind, just realized that it does not. Can we make the list of studies dependent on the selected species?
![image](https://user-images.githubusercontent.com/25048342/67709842-06642c80-f97c-11e9-9f2b-92dc14cb7cc9.png)

**Comment by karawoo** [comment]
I don't think there is one location where this information is tracked currently, so we would have to create it, and it would involve making another Synapse call that would slow things down.

**Comment by Aryllen** [comment]
Agreed. If we had the full table finished (and all studies had species annotations on their files), we could potentially use that to query for this, but it would slow down the app.
